### ISSUE TITLE
recreate a project from current deployed serverless AWS code  

### ISSUE THREAD

**Issue by engmsaleh** [serverless/serverless#5555]
I have current deployed serverless deployed to AWS, but I don't have the original serverless project, Is there any way to rebuild the serverless project from the current deployed code on My AWS?

**Comment by dschep** [comment]
Not currently. Sorry. I think the closest you'll get is AWS's builtin ability to download a function (dunno about more than one) as a SAM configuration.

**Comment by engmsaleh** [comment]
I think the complexity in the AWS resources, I'm trying to check the `compiled-cloudformation-template.json`, but it have a lot of things, I don't know if that is all required or serverless add some of its things their?

**Comment by engmsaleh** [comment]
The other thing I want to know, If I couldn't recreate the serverless project and wanted to move it from AWS account to another, Will I could be able to do so? and what is the required files to collect, export and import there?

**Comment by dschep** [comment]
Yeah, there's a lot in the compiled cfn template that is added automatically by the framework. CloudWatch stuff is usually all added automatically. Much of the ApiGateway stuff (if you have it) is generated from a rather minimal config in the `serverless.yml`

To move it to an other account, it should be possible if you ahve the cfn templates and the zip files in the s3 bucket that contain the handler code.
